### DISCUSSION TITLE
[BUG]WAF封锁记录解封问题

### DISCUSSION THREAD

**Discussion by sagezhj** [1Panel-dev/1Panel#4918] [Q&A]
### 联系方式

zhjah@163.com

### 1Panel 版本

v1.10.6-lts

### 问题描述

高级功能——WAF——封锁记录中，已解封再次被封锁地址仍然显示已解封，无法手动点击进行解封

### 重现步骤

尝试访问网站A超过设定访问频率，导致客户端地址被封锁。等待设定的解封时间自动解封后，再次以超过设定频率访问网站A，导致再次被封锁，此时，客户端处于被封锁状态，网站A无法访问，但解封记录里，客户端状态为已解封，解封按钮为浅色不可点击。

### 期待的正确结果

再次解封时，客户端地址显示被封锁，可以通过点击解封进行手动解封

### 附加信息

![未命名](https://github.com/1Panel-dev/1Panel/assets/22697674/ae06a27c-beed-4059-99b5-b36a20d695c9)
截图时间为16点35，按照预定策略，处于被封锁状态，实际访问确实仍被封锁，但是记录显示已解封

**Comment by zhengkunwang223** [comment]
我们先复现一下

**Comment by mjsoftking** [comment]
@zhengkunwang223 
![image](https://github.com/1Panel-dev/1Panel/assets/23021469/7a0473a3-f36f-4dd1-8012-806b05edaff4)
还有这种的没到解封时间，现在显示为已解封，未测试这种情况下是否可以访问

昨天晚上操作过开启“恶意 IP 组”操作，不知道是否有关

**Comment by zhengkunwang223** [comment]
> @zhengkunwang223 ![image](https://private-user-images.githubusercontent.com/23021469/328311794-7a0473a3-f36f-4dd1-8012-806b05edaff4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxNDg2MTcsIm5iZiI6MTcxNTE0ODMxNywicGF0aCI6Ii8yMzAyMTQ2OS8zMjgzMTE3OTQtN2EwNDczYTMtZjM2Zi00ZGQxLTgwMTItODA2YjA1ZWRhZmY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDA2MDUxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU3ZDRhMzc5NzA3OGQ3ZDkxZjJlOTZhMGQ0ZmNkODU1YTg4MDU5NDc0OWQ5NTgwYWY4ZGU3OTUyMThlNmExNjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.O7pZ09BTBQCt4m_4xh9EqJC9T2aICd8wqpdI0hBVDos) 还有这种的没到解封时间，现在显示为已解封，未测试这种情况下是否可以访问
> 
> 昨天晚上操作过开启“恶意 IP 组”操作，不知道是否有关

封锁 IP 临时存储在 OpenResty 中，重启 OpenResty 会解封，可以通过拉黑功能永久拉黑
